forumhelpers/FHP.js

```diff
@@ -6,6 +7,7 @@ function getData(dataSet) {
 			document.getElementById(dataSet+"Number").innerHTML = data.length;
 			for (j = 0; j < data.length; j++) {
 				var name = data[j].name;
+				membersList.push(name)
```

Comment:
a few places in the code, like this, have no semicolons as line terminators. is it possible to fix that?

Reply:
Yep, I can.